src/components/elements/windmill.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import {Entity} from "aframe-react";
 
 
-export default class Windmill extends React.Component {
```

Comment:
Why default was removed?

Reply:
I added there other models, then I removed them. I will bring it back.